sdktests/testapi_sdk_data_system.go

```diff
@@ -0,0 +1,47 @@
+package sdktests
+
+import (
+	"errors"
+
+	"github.com/launchdarkly/sdk-test-harness/v2/framework/opt"
+	o "github.com/launchdarkly/sdk-test-harness/v2/framework/opt"
+	"github.com/launchdarkly/sdk-test-harness/v2/servicedef"
+)
+
+type DataSystem struct {
```

Comment:
Again, I think we are going to see some different patterns we may want to emerge as we build out more tests. Just KISSing it for now.